cmd/benchdb/main.go

```diff
@@ -140,21 +141,21 @@ func (ut *benchDB) mustParseWork(work string) (name string, spec string) {
 func (ut *benchDB) mustParseInt(s string) int {
 	i, err := strconv.Atoi(s)
 	if err != nil {
-		log.Fatal(err)
+		log.Fatal(err.Error())
 	}
 	return i
 }
 
 func (ut *benchDB) mustParseRange(s string) (start, end int) {
 	strs := strings.Split(s, "_")
 	if len(strs) != 2 {
-		log.Fatal("invalid range " + s)
+		log.Fatal("parse range failed", zap.String("invalid range", s))
```

Comment:
Should we use `invalidRange`?

Reply:
I think it's OK. `s` isn't a variable named `invalidRange`.